cmd/acra-server/listener.go

```diff
@@ -402,6 +410,11 @@ func (server *SServer) StartCommands() {
 func (server *SServer) StartCommandsFromFileDescriptor(fd uintptr) {
 	var connection net.Conn
 	file := os.NewFile(fd, "/tmp/acra-server_http_api")
+	if file == nil {
+		log.Errorln("can't create new file from descriptor for api listener")
```

Comment:
usually we start logs with capital letter :)

just saying

Reply:
will update. forgot